include/Encode/LegacyEncodingManager.h

```diff
@@ -3,6 +3,23 @@
 #include <Encode/EncodingManager.h>
 #include "ForceFeedback.h"
 
+enum class VRCommDataLegacyEncodingPosition : int {
+  FIN_THUMB,
+  FIN_INDEX,
+  FIN_MIDDLE,
+  FIN_RING,
+  FIN_PINKY,
+  JOY_X,
+  JOY_Y,
+  JOY_BTN,
+  BTN_TRG,
+  BTN_A,
+  BTN_B,
+  GES_GRAB,
+  GES_PINCH,
+  MAX,
+};
+
```

Comment:
Likewise, I don't think this is used anywhere else

Reply:
Fixed with latest commit